paradise_papers_search/paradise_papers_search/settings/production.py

```diff
@@ -0,0 +1,14 @@
+from .base import *
+
+# SECURITY WARNING: don't run with debug turned on in production!
+DEBUG = True
```

Comment:
Comment to do not run the server with debug turned on in prod and the settings have it turned on 🤔  Interesting   😂  Also, change the comment to be: `don't run productions with debugging turned on` I think it's better worded that way.

Reply:
Nice catch!! Forgot to update this 😅